mkt/extensions/indexers.py

```diff
@@ -113,7 +120,9 @@ def extract_document(cls, pk=None, obj=None):
         else:
             doc['latest_public_version'] = None
         doc['name_sort'] = unicode(obj.name).lower()
-        doc['guid'] = unicode(obj.uuid)
+        # Find the first reviewed date (used in sort).
```

Comment:
Is the sort here "first time approved in Marketplace"? That's not particularly intuitive.